sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -10,23 +10,71 @@
 
 namespace Azure.Core
 {
+    /// <summary>
+    /// Represents the content of sent as part of <see cref="Request"/>
```

Comment:
Did you mean "... content sent as part of the <see cref="Request" />" ?

nit:  Missing period at the end.